src/verify_cert.rs

```diff
@@ -172,6 +175,80 @@ impl<'a> ChainOptions<'a> {
     }
 }
 
+/// Path from end-entity certificate to trust anchor that's been verified.
```

Comment:
I think this might be a helpful place to point back to `verify_trusted` to give more details on what verified means.